Makefile

```diff
@@ -112,7 +113,23 @@ installer:
 		--frontend-opt target=$@ \
 		$(COMMON_ARGS)
 	@docker load < build/$@.tar
-	@docker run --rm -v /dev:/dev -v $(PWD)/build:/out --privileged autonomy/$@:$(TAG) image -l
+
+do-image: installer
```

Comment:
three new make commands, one for each img type.

Reply:
```suggestion
image-do: installer
```